modules/new_serverpod_auth/serverpod_auth_jwt/serverpod_auth_jwt_server/lib/src/business/jwt_util.dart

```diff
@@ -79,7 +79,7 @@ class JwtUtil {
 
     final UuidValue refreshTokenId;
     try {
-      refreshTokenId = UuidValue.fromString(jwt.jwtId!);
+      refreshTokenId = UuidValue.withValidation(jwt.jwtId!);
```

Comment:
The only other (non test) place we use `UuidValue.fromString` is for any default model values.

Should we update the code generator here, check the value once during code gen, or leave it as is (in which case the DB write might fail)? 

https://github.com/serverpod/serverpod/blob/b7fd2625281e112dfbbbb2889270cbe8e9c7a97b/tests/serverpod_test_client/lib/src/protocol/defaults/uuid/uuid_default_mix.dart#L22

Reply:
Ah, perfect! :)